src/main/java/edu/usf/cutr/gtfsrtvalidator/helper/TimeStampHelper.java

```diff
@@ -18,9 +18,10 @@
 package edu.usf.cutr.gtfsrtvalidator.helper;
 
 import java.util.Date;
+import java.util.concurrent.TimeUnit;
 
 public class TimeStampHelper {
     public static long getCurrentTimestamp() {
-        return (long) Math.ceil((double) new Date().getTime() / 1000);
+        return TimeUnit.MILLISECONDS.toSeconds(new Date().getTime());
```

Comment:
Nice job using `TimeUnit.MILLISECONDS.toSeconds()`!  We should really use millisecond precision for timestamps and use `System.currentTimeMillis()` instead of instantiating a new Date object, but let's handle that as part of #133.  I just commented on this at https://github.com/CUTR-at-USF/gtfs-realtime-validator/issues/133#issuecomment-294488136.